praw/models/reddit/mixins/inboxable.py

```diff
@@ -101,6 +101,33 @@ def mark_unread(self):
         """
         self._reddit.inbox.mark_unread([self])
 
+    def unblock_subreddit(self):
```

Comment:
Does this work for unblocking users as well? I ask because it might be better to name this `unblock` and just switch between endpoints depending on the object kind.

Reply:
That's an interesting idea. I just tried it, and it doesn't seem to work.

Reply:
What endpoints are you using?

Reply:
To unblock a sub, you send `{"id": "t4_12345"}` as the payload, where `t4_12345` is the fullname of a message sent to you by that subreddit. I don't know why there isn't a list of blocked subreddits like there is a list of blocked users, but that's a different issue.

You can block a user via the inbox with api/block by sending the fullname of the message or comment reply. I don't know of a way of unblocking a user that uses the fullname of a message in your inbox. I blocked someone from a comment reply and a private message using api/block, and then tried using api/unblock_subreddit with the id set to the fullname of the message I blocked, but it didn't unblock them.

I think there is a separate endpoint for unblocking a subreddit because subreddit messages are different than other messages. If you block a user, their messages disappear from your inbox until you unblock them. If you block a subreddit, the messages stay in your inbox, but the subject of the message changes, which is shown in the test.

Reply:
Ah I see. It was just a thought, I was curious if you could unblock a user using their username or account fullname from a message from them but if their messages disappear from the inbox it wouldn't be useful to also have this method unblock users as well.